tangelo/linq/tests/test_simulator_noisy.py

```diff
@@ -65,6 +65,11 @@ def test_noisy_simulation_not_supported(self):
             Ensures that an error is returned if user attempts to run noisy simulation on a backend that does
             not support it as part of this package.
         """
+        backend_info = dict()
```

Comment:
I am confused. Why is this information hardcoded into a test file ?
Is there a bug or circular dependency preventing us from using `backend_info` ?